android/sdl_android/src/main/java/com/smartdevicelink/proxy/SdlProxyBase.java

```diff
@@ -629,10 +633,11 @@ public void onProtocolSessionStarted(SessionType sessionType,
 					 
 					startRPCProtocolSession();
 				}
-				else

```

Comment:
Removing the else will break existing functionality.

Reply:
Thanks Markos, I should be putting this callback back.